### PR TITLE
Only allow daemon to be run as root if --as-root flag is passed

### PR DESCRIPTION
**Pull request by lezed1**
https://trello.com/c/JjryMZdY/50-check-that-the-daemon-isnt-started-with-sudo

### CODE REVIEW

File: daemon/run_openrazer_daemon.py [hunk 1]
```diff
@@ -43,6 +43,8 @@ def parse_args():
     parser.add_argument('-s', '--stop', action='store_true', help='Gracefully stop the existing daemon.')
     parser.add_argument('--version', action='version', version='%(prog)s {version}'.format(version=__version__))
 
+    parser.add_argument('--as-root', action='store_true', help='All the daemon to be started as root')
```
Comment (z3ntu) [inline daemon/run_openrazer_daemon.py]: help text seems wrong

**Review by z3ntu — CHANGES_REQUESTED** [review]
Also please add the new flag to the man page

**Comment by z3ntu** [comment]
And add the as-root flag to the file scripts/ci/launch-daemon.sh to fix the CI

**Comment by lezed1** [comment]
I didn't expect the new flag to be needed so quickly... All your feedback should be addressed!

**Comment by z3ntu** [comment]
Thanks.

**Review by z3ntu — APPROVED** [review]
LGTM

**Review by terricain — APPROVED** [review]

**Pull request merged**